plugins/rollup-plugin-remove-html-elements.js

```diff
@@ -0,0 +1,37 @@
+// rollup-plugin-remove-html-elements.js
```

Comment:
A simple config to remove all the htmlelements from the final build.
Note: This is currently changing real code as well. Will need to fix it